docs/ruleset_functions/is_terminal_supply_ducted(terminal.id).md

```diff
@@ -0,0 +1,21 @@
+# is_terminal_supply_ducted
+
+**Description:** Returns TRUE if the terminal supply is ducted. It returns FALSE if the terminal is supply ducted data element equals FALSE.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the terminal supply is ducted.    
+- **terminal_b.id**: The id of the terminal unit to evaluate.  
+
+**Returns:**  
+- **is_terminal_supply_ducted**: The function returns TRUE if the terminal supply is ducted. It returns FALSE if the terminal is supply ducted data element equals FALSE.      
```

Comment:
I wonder if this element has any impact on simulation? 

Also, as with all terminal-related sub-functions, recommend looping through all terminals associated with a given HVAC.

Reply:
Discussed on call and via email. I think we decided to keep it.

Update terminal looping.